lib/shelley/src/Cardano/Wallet/Shelley/Launch.hs

```diff
@@ -777,19 +796,37 @@ genConfig
     -- ^ A top-level directory where to put the configuration.
     -> Severity
     -- ^ Minimum severity level for logging
+    -> Maybe (FilePath, Severity)
+    -- ^ Optional /extra/ logging output
     -> UTCTime
     -- ^ Genesis block start time
     -> IO (FilePath, Block, NetworkParameters, NodeVersionData)
-genConfig dir severity systemStart = do
+genConfig dir severity mExtraLogFile systemStart = do
     let startTime = round @_ @Int . utcTimeToPOSIXSeconds $ systemStart
     let systemStart' = posixSecondsToUTCTime . fromRational . toRational $ startTime
 
+    let fileScribe (path, sev) = ScribeDefinition
+                { scName = path
+                , scFormat = ScText
+                , scKind = FileSK
+                , scMinSev = sev
+                , scMaxSev = Critical
+                , scPrivacy = ScPublic
+                , scRotation = Nothing
+                }
+
+    let scribes = catMaybes
+            [ Just $ fileScribe ("cardano-node.log", severity)
+            , fileScribe . first T.pack <$> mExtraLogFile
+            ]
```

Comment:
What's the use for two log files here?

Reply:
It seemed drastic/weird to remove the existing node logs from the temporary directory.

Without setting additional environment variables, we can inspect the dbs, and the logs, in that directory, while the tests are running.

Thought it is also a bit weird to duplicate the logs 🤷‍♂️